packages/manager/modules/vps/src/upscale/upscale.controller.js

```diff
@@ -364,12 +370,13 @@ export default class UpscaleController {
     return [parseInt(cores, 10), parseInt(memory, 10), parseInt(storage, 10)];
   }
 
-  setValueIfUniqueChoice(values, path) {
+  initRangeConfiguration(values, path) {
     if (values.length === 1) {
       [this.rangeConfiguration[path]] = values;
     }
```

Comment:
I think you can remove this as it is done everytime on next line

Reply:
You right in fact i was inspired by a previous PR close where there was indeed this inconsistency.
It's fixed.